crates/sound/src/audio_backends.rs

```diff
@@ -47,12 +45,10 @@ pub trait AudioBackend {
 
 pub fn alloc_audio_backend(
     backend: BackendType,
-    // Unused when compiled with no features.
-    #[allow(unused_variables)] streams: Arc<RwLock<Vec<Stream>>>,
+    streams: Arc<RwLock<Vec<Stream>>>,
 ) -> Result<Box<dyn AudioBackend + Send + Sync>> {
     log::trace!("allocating audio backend {:?}", backend);
     match backend {
-        #[cfg(feature = "null-backend")]
```

Comment:
Can we now remove also the `_` pattern?

Reply:
Ah you are right. At first glance as I was doing this I thought it was still required.
On second thought though, it is not. I will remove it.

Thanks!